deepchem/models/torch_models/ferminet.py

```diff
@@ -0,0 +1,127 @@
+"""
+Implementation of the Ferminet class in pytorch
+"""
+
+from typing import List, Optional
+# import torch.nn as nn
+from rdkit import Chem
+import numpy as np
+
+# from deepchem.models.torch_models import TorchModel
+# import deepchem.models.optimizers as optim
+from deepchem.utils.electron_sampler import ElectronSampler
+
+# TODO look for the loss function(Hamiltonian)
+
+
+class Ferminet:
+  """A deep-learning based Variational Monte Carlo method for calculating the ab-initio
+    solution of a many-electron system.
+
+    This model aims to calculate the ground state energy of a multi-electron system
+    using a baseline solution as the Hartree-Fock. An MCMC technique is used to sample
+    electrons and DNNs are used to caluclate the square magnitude of the wavefunction,
+    in which electron-electron repulsions also are included in the calculation(in the
+    form of Jastrow factor envelopes). The model requires only the nucleus' coordinates
+    as input.
+
+    This method is based on the following paper:
+
+    Spencer, James S., et al. Better, Faster Fermionic Neural Networks. arXiv:2011.07125,
+    arXiv, 13 Nov. 2020. arXiv.org, http://arxiv.org/abs/2011.07125.
+    """
+
+  def __init__(self,
+               nucleon_coordinates: List[List],
+               seed_no: Optional[int] = None,
+               batch_number: int = 10):
```

Comment:
ElectronSampler calls these arguments `seed` and `batch_no`.  We should make the two classes consistent.

Reply:
I didn't quite understand the "class consistency" here 😅. Did u mean like I should remove the seed_no and batch_number here in ferminet class? If so, then how will the user set the seed and batch number then?

Reply:
I think he means the name of the arguments should be changed from `seed_no` to `seed` and `batch_number` to `batch_no`.